website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -207,12 +207,14 @@ const useExternalApiFetcher = () => {
     context: createApolloContextForProxyForwardTo({
       // The URL to your external API
       uri: externalApiUrl,
-      // Optionally pass custom HTTP headers
+      // Provide custom HTTP headers (optional)
       headers: {
         'x-foo': 'bar',
       },
-      // Optionally define the audience policy
+      // Define the audience policy (optional)
       audiencePolicy: 'forward-url-full-path',
+      // Ask for users permissions to be included in exchange JWT (optional)
```

Comment:
```suggestion
      // Set `"X-Forward-To-Claims": "permissions"` header in the request (optional).
```

Reply:
@Anshuman71 if we include the header name in the comment, I'm wondering if we should also change the comment in the line above about the `audiencePolicy` property (which includes the `X-Forward-To-Audience-Policy` header in the request) to have a consistent messaging.
Otherwise, we would have something like this:
```
// Optionally define the audience policy
audiencePolicy: 'forward-url-full-path',
// Set `"X-Forward-To-Claims": "permissions"` header in the request (optional)
includeUserPermissions: true
```

Should we use that format or should we make both comments similar?
```
// Set `"X-Forward-To-Audience-Policy"` header in the request with provided value (optional)
audiencePolicy: 'forward-url-full-path',
// Set `"X-Forward-To-Claims": "permissions"` header in the request (optional)
includeUserPermissions: true
```

Reply:
That sounds good. This is a good practice when the options are explained in the docs below. The comments should only say what happened in that line and readers should fall back to the docs for more information.

Reply:
Updated in this commit: c8053312999955e37e23bc1c73a9fb8896af5314